Vanilla/jsconnect.cs

```diff
@@ -155,7 +155,7 @@ public class jsConnect {
 				if(sigStr.Length > 0)
 					sigStr.Append("&");
 				
-				string encValue = jsConnect.UrlEncode(data[key].ToString());
+				string encValue = jsConnect.UrlEncode(data[key]?.ToString());
```

Comment:
The "?." operator is new in C#. It would be our preference that you use something compatible with older versions of C#.